README.md

```diff
@@ -37,21 +36,95 @@ Deployment of NVA consists of 2 templates which is used to create resources in 2
-| |nva-cognito-post-authentication-trigger|NvaCognitoPostAuthenticationTrigger|0.1.1|2020-09-09| |
+***
+
+## Deployment steps
+
+ 1. Identify repositories with changes.
+ 2. Merge develop into master for each repositor with changes.
+ 3. Create new tag in repository.
+ 4. Verify CodePipeline for publishing to SAR (Serverless Repo) has run successfully by checking versions and and date in SAR.
+ 5. Update deploy templates with new applications, versions and paramters (if any), see above.
+ 6. Verify that any external resources has been updates (DynamoDB, Secrets, Elasticsearch, Cognito etc).
+ 7. Deploy templates by updating stacks in Cloudformation.
+ 8. Verify changes to stack is Completed.
+ 9. Deployment is done.
+
+## NVA Applications Overview
+
+Overview of Serverless Applications in the NVA platform.
+
+### Frontend
+
+**[Frontend](https://github.com/BIBSYSDEV/NVA-frontend)**
+
+Frontend for the NVA application.
+
+### APIs
+
+These applications use DynamoDB tables. Ensure all tables and indices are up to date before updating
+application itself.
+
+**[Publication API](https://github.com/BIBSYSDEV/nva-publication-api)**
+
+For managing Publications.
+
+**[Customers API](https://github.com/BIBSYSDEV/nva-customer-api)**
+
+For managing Customer Instititions.
+
+**[DOI Requests API](https://github.com/BIBSYSDEV/nva-doi-requests-api)**
+
+For managing Publications DOI Requests to Datacite.
+
+**[Users and Roles Service](https://github.com/BIBSYSDEV/nva-user-access-service)**
+
+For managing users of NVA and their roles. Uses API Key stored as Secret in AWS to protect parts of 
+the API.
+
+**[Upload Multipart](https://github.com/BIBSYSDEV/nva-upload-multipart)**
+
+For uploading files to Publications.
+
+**[Download Publication File API](https://github.com/BIBSYSDEV/nva-download-file)**
+
+For downloading files from Publications.
+
+### Triggers
+
+**[Cognito Post Authentication Trigger](https://github.com/BIBSYSDEV/nva-cognito-post-authentication-trigger)**
+
+For looking up existing authenticated users or create default users for new logins. Uses API Key 
+stored as Secret in AWS to communicate with Users and Role Service.
+
+### Proxies to external services
+
+**[Fetch DOI](https://github.com/BIBSYSDEV/nva-fetch-doi)**
+
+For lookup up metadata (Dataciter, Crosstef) for DOI. Metadata found is transformed to Publication and stored in NVA.
+
+**[SRU Last Publication](https://github.com/BIBSYSDEV/nva-alma-proxy)**
+
+???
+
+**[Person Data](https://github.com/BIBSYSDEV/nva-bare-proxy)**
+
+For looking up Authority metadata for Person.
+
+**[Publication Channel Register](https://github.com/BIBSYSDEV/nva-channel-registry)**
+
+For looking up Publication Channels.
+
+**[Projects](https://github.com/BIBSYSDEV/nva-cristin-projects)**
+
+For looking up Projects in Cristin.
+
+**[Institution Proxy](https://github.com/BIBSYSDEV/nva-instituion-proxy)**
+
+For looking up Institutions in Cristin.
```

Comment:
For viewing structured information about Organisations in Cristin? Basically, this service provides a top-level list of Norwegian research Organisations and provides structured hierarchical views of departments in these.